type_classes.md

```diff
@@ -59,19 +59,56 @@ class Add (a : Type) where
 where the square brackets indicate that the argument of type `Add a` is *instance implicit*,
 i.e. that it should be synthesized using typeclass resolution. This version of
 `add` is the Lean analogue of the Haskell term `add :: Add a => a -> a -> a`.
-Similarly, we can register an instance by
+Similarly, we can register instances by
 ```lean
 # namespace Ex
 # class Add (a : Type) where
 #  add : a -> a -> a
 instance : Add Nat where
   add := Nat.add
 
+instance : Add Int where
+  add := Int.add
+
+instance : Add Float where
+  add := Float.add
+
+# end Ex
+```
+Then for `n : Nat` and `m : Nat`, the term `Add.add n m` triggers typeclass resolution with
+the goal of `Add Nat`, and typeclass resolution will synthesize the instance for `Nat` above.
+We can now reimplement `double` using an instance implicit by
```

Comment:
Is there a tendency to prefer passive voice or second person to first person plural? (ie "you" instead of "we", etc). A merged PR had some rewrites of this, I noticed. Happy to change if that's the preferred style.